wgpu-hal/src/vulkan/command.rs

```diff
@@ -747,6 +772,18 @@ impl crate::CommandEncoder for super::CommandEncoder {
 
         for cat in desc.color_attachments {
             if let Some(cat) = cat.as_ref() {
+                let color_view = if cat.target.view.dimension == wgt::TextureViewDimension::D3 {
+                    let key = super::TempTextureViewKey {
+                        texture: cat.target.view.raw_texture,
+                        format: cat.target.view.raw_format,
+                        mip_level: cat.target.view.base_mip_level,
+                        depth_slice: cat.depth_slice.unwrap(),
+                    };
+                    self.make_temp_texture_view(key)?
+                } else {
+                    cat.target.view.raw
+                };
+
```

Comment:
question: Is destructuring here interesting for more refactor-proof code?

Reply:
I don't think so as we don't use the view's `layers` of `format` and if we add other fields in the future they most likely don't have to be used here either.